rqt_py_common/src/rqt_py_common/message_field_type_helpers.py

```diff
@@ -0,0 +1,407 @@
+      -----------------------------------------------------------
+      bool            -> 'boolean'                  -> bool
+      byte            -> 'octet'                    -> bytes
+      float32         -> 'float'                    -> float
+      float64         -> 'double'                   -> float
+      char            -> 'uint8'                    -> int
+      int8            -> 'int8'                     -> int
+      int16           -> 'int16'                    -> int
+      int32           -> 'int32'                    -> int
+      int64           -> 'int64'                    -> int
+      uint8           -> 'uint8'                    -> int
+      uint16          -> 'uint16'                   -> int
+      uint32          -> 'uint32'                   -> int
+      uint64          -> 'uint64'                   -> int
+      string          -> 'string'                   -> str
+      my_msgs/Custom  -> 'my_msgs/Custom'           -> my_msgs.msg.Custom
+
+      # Arrays with normal values
+      int8[3]         -> 'int8[3]'                  -> List[int]
+      int8[]          -> 'sequence<int8>'           -> List[int]
+      int8[<=3]       -> 'sequence<int8, 3>'        -> List[int]
+
+      # String weirdness
+      string<=5       -> 'string<5>'                -> str
+      string[3]       -> 'string[3]'                -> List[str]
+      string[]        -> 'sequence<string>'         -> List[str]
+      string<=5[3]    -> 'string<5>[3]'             -> List[str]
+      string<=5[<=10] -> 'sequence<string<5>, 10>'  -> List[str]
+      string<=5[]     -> 'sequence<string<5>>'      -> List[str]
+      string[<=10]    -> 'sequence<string, 10>'     -> List[str]
+
+    :param field_type: the field type in the form generated by msg.get_fields_and_field_types
+    :type field_type: str
+    '''
+
+    def __init__(self, field_type: str):
+        super(MessageFieldTypeInfo, self).__init__()
+
+        self.base_type_str = base_type_str(field_type)
+        self.is_array = is_array(field_type)
+        self.is_static_array = is_static_array(field_type)
+        self.static_array_size = static_array_size(field_type)
+        self.is_bounded_array = is_bounded_array(field_type)
+        self.bounded_array_size = bounded_array_size(field_type)
+        self.is_unbounded_array = is_unbounded_array(field_type)
+        self.is_bounded_string = is_bounded_string(field_type)
+        self.bounded_string_size = bounded_string_size(field_type)
+
+    def get_field_type_info_as_dict(self) -> Dict[str, Any]:
+        '''Returns the type info as a dictionary'''
+        info_as_dict = {}
+        for slot in MessageFieldTypeInfo.__slots__:
+            info_as_dict[slot] = getattr(self, slot)
+        return info_as_dict
+
+
+__END_OF_BASE_TYPE_DELIM = ['<', '[', ',', '>']
+
+
+def base_type_str(field_type: str) -> str:
+    '''
+    The base type of the field_type (eg, uint8, my_msgs/Custom, ...)
```

Comment:
In general, according to the pep257, docstring function comments should use the imperative, fix here and below